### DISCUSSION TITLE
When will inverted index feature public release?

### DISCUSSION THREAD

**Discussion by uranuswch** [ClickHouse/ClickHouse#48748] [Q&A] [answered]
I have seen this doc [inverted indexes](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/invertedindexes), and my product have a strong dependency on full-text search, so I want to know about the inverted index release timeline, please tell me if have any plan or timeline, many thanks.

**Comment by den-crane** [comment, marked as answer]
https://github.com/ClickHouse/ClickHouse/blob/master/CHANGELOG.md#-clickhouse-release-231-2023-01-26
ClickHouse release 23.1, 2023-01-26
Add an experimental inverted index as a new secondary index type for efficient text search. #38667 (larryluogit).

**Reply by uranuswch** [reply]
yeah, I have use this experimental feature in my test env, but if this change in the future or not? so I want to know about the roadmap of full-text search.

**Reply by den-crane** [reply]
There is no exact plans. Eventually or never.

**Reply by uranuswch** [reply]
Thanks for reply.
